### PR TITLE
changed typecasting from np.float to float  in calc_loss_function

### PR DESCRIPTION
**Pull request by ritog**
This PR solves Issue #1168 

#### Change highlight:
```
# @markdown Execute this cell to enable the function `calc_loss_func`

def calc_loss_func(loss_f, mu_true, x):
    # ...
    elif loss_f == "Zero-One Loss":
        loss = (np.abs(error) >= 0.03).astype(float)
    return loss
    
```

Changed from `astype(np.float)` to `astype(float)` in the function `calc_loss_func`.

This enables the interactive demo to show the plots for Zero-One Loss which was not possible before, as pointed out in the issue.

image

### CODE REVIEW

**Comment by ritog** [comment]
Made changes as per: https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated

**Review by iamzoltan — APPROVED** [review]
lgtm
